docker-compose.ci.yml

```diff
@@ -75,6 +75,7 @@ services:
         volumes:
             - ./build:/srv/annotations/build
             - ./config/container.php:/srv/annotations/config.php
+            - .php_cs.cache:/srv/annotations/.php_cs.cache
```

Comment:
Can't reproduce the failure locally yet, there may be some state in the workspace. I suspect however that if this is a single file, when it does not exist it is created as an empty folder which is not what we want?

Reply:
My local build stops at
```
PHP-CS-Fixer consistency check with local project: /srv/proofreader-php/.php_cs vs .php_cs
Discrepancy between /srv/proofreader-php/.php_cs and .php_cs: cannot continue
--- Original
+++ New
@@ @@
   ),
-   'usingCache' => false,
+   'usingCache' => true,
    'usingLinter' => true,
    'hideProgress' => false,
 ))
```

Reply:
ah, the `elife-libraries--ci` server must be using an old version of the `proofreader` image. Interesting

Reply:
(or I must be)

Reply:
But if we do `docker-compose build --pull` the `ci` image fails to build because it attempts to pull `elifesciences/annotations_cli:$commit` to extend it; but we don't push all images to Docker Hub. Better to pin the `proofreader-php` image.

Reply:
Finally I get the same result. Now, is there a way to move the `.php_cs.cache` location so that we can mount a folder rather than a file that usually does not exist yet?

Reply:
Inside the container? We'll need to move to PHP-CS-Fixer 2.

Reply:
Brought up https://elifesciences.atlassian.net/browse/ELPP-3163 in priority but this PR should still go ahead without the cache to get the rest of the benefits.